department-of-veterans-affairs/caseflow
# Issue 7773 - Fixes for user demo.

Resolves #7773 

### Description
Fixed error message in modal and added seed data.

### Acceptance Criteria

### Testing Plan
1. Go to assign hearings page
2. Confirm the number of slots available is correct given the new number of rows in db for central office dates.
